packages/terraform-modules/testnet/main.tf

```diff
@@ -0,0 +1,92 @@
+provider "google" {
+  credentials = file("~/.gcloud/service-accounts/celo-testnet-69a4cd9692dd.json")
```

Comment:
This will be fixed in a later PR

Reply:
Not 100% sure, but would https://github.com/celo-org/celo-monorepo/blob/master/packages/celotool/src/lib/gcloud_utils.ts#L20 possibly help addressing this?